README.md

```diff
@@ -8,6 +8,74 @@ Subscription Cleanup Job is a subcomponent of [Kyma Control Plane](https://githu
 
 For more information on KCP and its components, read the [KCP documentation](https://github.com/kyma-project/control-plane/tree/main/docs) and the [KEB documentation](https://github.com/kyma-project/kyma-environment-broker/blob/main/docs).
 
+## Runnining locally
+
+- download your Gardener Kubeconfig (personal kubeconfig, service account isn't needed)
+- build subscription cleanup job: `go build ./cmd/subscriptioncleanup/main.go`
+- set environmental variables:
+  - `APP_GARDENER_PROJECT=frog-dev`
+  - `APP_GARDENER_KUBECONFIG_PATH=$PWD/kubeconfig-garden-frog-dev.yaml`
+- run the job: `./main`
+
+SCJ will look for secret bindings in the Gardener with `dirty=true` and `hyperscalerType` labels.
+If `hyperscalerType` is not set, the secret will be logged and ignored.
```

Comment:
```suggestion
If `hyperscalerType` is not set, the Secret is logged and ignored.
```

Reply:
As this is a rule, not a prediction, please do not use the future form. See the [content guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#tenses).